build/deploy/main.go

```diff
@@ -0,0 +1,122 @@
+// main handles deployment of the plugin to a development server using either the Client4 API
```

Comment:
Package main handles ...

Reply:
`build/` is part of the files that get synced with https://github.com/mattermost/mattermost-plugin-starter-template. To me consistency is more important then correctness, hence I'm leaning towards asking you to report this on https://github.com/mattermost/mattermost-plugin-starter-template-